static/js/settings_org.js

```diff
@@ -182,7 +190,10 @@ function get_property_value(property_name) {
     return page_params[property_name];
 }
 
-export function extract_property_name(elem) {
+export function extract_property_name(elem, for_realm_default_settings) {
+    if (for_realm_default_settings) {
+        return elem.attr("name");
+    }
```

Comment:
Have used `name` attribute since we do not have id for those elements for now and if we want to add we would need some workaround for emojiset setting since it is a single setting with four different input elements.

Reply:
I think this is probably reasonable; maybe worth adding a comment explaining the background there.